modules/terraform/output.go

```diff
@@ -49,6 +50,45 @@ func OutputRequiredE(t *testing.T, options *Options, key string) (string, error)
 	return out, nil
 }
 
+// OutputList calls terraform output for the given variable and return its value as a list.
+// If the output is a single value, then it returns a list with just one item.
+func OutputList(t *testing.T, options *Options, key string) []string {
+	out, err := OutputListE(t, options, key)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return out
+}
+
+// OutputListE calls terraform output for the given variable and return its value as a list.
+// If the output is a single value, then it returns a list with just one item.
```

Comment:
If it's a non-list value, shouldn't this return an error instead? It seems like that might indicate a bug with types or a typo in the variable name...

Reply:
I was thinking about throwing an error too. It's a design decision and I don't have strong opinion here. I'll change it to throwing the error then.